front/pages/auth/signup/success.vue

```diff
@@ -0,0 +1,232 @@
+<template>
+  <div id="signupSuccessPage">
+    <v-layout row class="gridContainer">
+      <v-flex class="lefgImgContainer" hidden-sm-and-down>
+        <img
+          class="imgPersonWithPhone"
+          src="~/assets/images/person_with_phone.png"
+        />
+      </v-flex>
+
+      <!-- Register Div -->
+      <v-flex class="successMessage">
+        <!-- Header Images -->
+        <img class="logoIcon" src="~/assets/images/signup/dark_gray_lamp.png" />
+        <br />
+        <img class="logoText" src="~/assets/images/logo_text.png" />
+
+        <!-- Hero Message -->
+        <div class="heroMsg">
+          Yay!
+        </div>
+
+        <!-- Hero Description -->
+        <div class="heroDescription">
+          Welcome to Daily Idea!<br />
+          We've sent you an email confirmation link
+        </div>
+
+        <!-- Continue Button -->
+        <v-btn large class="okBtn">OK</v-btn>
+
+        <!-- Social Login Icons -->
+        <div class="socialIconContainer">
+          <v-btn small outline fab color="primary">
+            <v-icon>fab fa-facebook-f</v-icon>
+          </v-btn>
+          <v-btn small outline fab color="primary">
+            <v-icon>fab fa-twitter</v-icon>
+          </v-btn>
+          <v-btn small outline fab color="primary">
+            <v-icon>fab fa-google-plus-g</v-icon>
+          </v-btn>
+        </div>
+      </v-flex>
+
+      <v-flex class="rightImgContainer" hidden-sm-and-down>
+        <img
+          class="imgLightGrayLamp"
+          src="~/assets/images/signup/light_gray_lamp.png"
+        />
+        <img
+          class="imgPersonWithPhone"
+          src="~/assets/images/signup/lady_with_phone.png"
+        />
+      </v-flex>
+    </v-layout>
+
+    <!-- Fixed Footer -->
+    <v-layout
+      hidden-sm-and-down
+      class="fixedFooter"
+      :style="{
+        'background-image':
+          'url(' + require('~/assets/images/signup/footer_background.png') + ')'
+      }"
+    >
+    </v-layout>
+  </div>
+</template>
+
+<script>
+import nanoid from 'nanoid'
+import { getErrorMessage } from '~/utils'
+export default {
+  data: () => ({
+    email: '',
+    success: false
+  }),
+  methods: {
+    async signup() {
+      try {
+        await this.$store.dispatch('cognito/registerUser', {
+          username: this.email,
+          password: nanoid()
+        })
+        this.success = true
+      } catch (e) {
+        this.$snotify.error(getErrorMessage(e), 'Error', {
+          timeout: 2000,
+          showProgressBar: false,
+          closeOnClick: true,
+          pauseOnHover: true
+        })
+      }
+    }
+  }
+}
+</script>
+
+<style lang="scss">
+@import './assets/style/variables.scss';
```

Comment:
You dont need to import variables at every file.
You can do it via loaders
https://nuxtjs.org/api/configuration-build#loaders
sass: {
        data: `@import "~@/assets/style/variables.scss"`,
      },

Reply:
Okay, Will try this

Reply:
ITs done.